platform/api-server/api/routes.js

```diff
@@ -41,6 +45,9 @@ module.exports = function(app) {
         gatewayAPIController.retrievePrivacyPolicy);
     app.get('/platform/link-graph-data', linkGraphController.getLinkGraphData);
     app.get('/platform/link-graph-visual', linkGraphController.renderLinkGraph);
+    app.post('/platform/execute-app',
```

Comment:
just a minor thing. can you change this to `platform/schedule-app` instead of `platform/execute-app`?